cypress/utils/user-state.ts

```diff
@@ -13,7 +14,12 @@ export const DISCONNECTED_WALLET_USER_STATE: Partial<UserState> = { selectedWall
 export function setInitialUserState(win: Cypress.AUTWindow, state: UserState) {
   // Selected wallet should also be reflected in localStorage, so that eager connections work.
   if (state.selectedWallet) {
-    win.localStorage.setItem('selected_wallet', state.selectedWallet)
+    win.localStorage.setItem(
```

Comment:
could we use the `setConnectionMeta` fn here?

Reply:
we can't without passing the window object, which complicates the setter. I chose to just export the key, rather than make the setter more complex.